### PR TITLE
dev-vcs/git-remote-hg: bump version to 1.0.1

### PR DESCRIPTION
**Pull request by realnc**
setup.py was fixed upstream. Patch no longer needed. Delete the patch and 1.0.0, as 1.0.1 is mostly a bugfix release for setup.py.

### CODE REVIEW

**Comment by realnc** [comment]
Ping.

File: dev-vcs/git-remote-hg/git-remote-hg-1.0.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+SRC_URI="https://github.com/mnauw/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+BDEPEND="
+	app-text/asciidoc
+"
+DEPEND="${PYTHON_DEPEND}"
```
Comment (mgorny) [inline dev-vcs/git-remote-hg/git-remote-hg-1.0.1.ebuild]: Isn't this redundant with distutils-r1?

File: dev-vcs/git-remote-hg/git-remote-hg-1.0.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+RDEPEND="
+	${DEPEND}
+	dev-vcs/git
+	dev-vcs/mercurial
+"
+
+# Some tests fail.
```
Comment (mgorny) [inline dev-vcs/git-remote-hg/git-remote-hg-1.0.1.ebuild]: Link upstream report.

File: dev-vcs/git-remote-hg/git-remote-hg-1.0.1.ebuild (hunk 1 above)
Comment (realnc) [reply dev-vcs/git-remote-hg/git-remote-hg-1.0.1.ebuild]: Indeed it is. Removed.

File: dev-vcs/git-remote-hg/git-remote-hg-1.0.1.ebuild (hunk 2 above)
Comment (realnc) [reply dev-vcs/git-remote-hg/git-remote-hg-1.0.1.ebuild]: There is none. However, the test suite prints this:

`TODO known breakage`

For tests that fail. So upstream is aware.

**Pull request closed without merging**